backend/resources/gpml/config.edn

```diff
@@ -447,7 +447,10 @@
   :gpml.handler.stakeholder/patch {:db #ig/ref :duct.database/sql}
   :gpml.handler.stakeholder/patch-params {}
   :gpml.handler.stakeholder/profile {:db #ig/ref :duct.database/sql}
-  :gpml.handler.stakeholder/get {:db #ig/ref :duct.database/sql}
+  :gpml.handler.stakeholder/get {:db #ig/ref :duct.database/sql
+                                 :auth0 {:domain #duct/env ["OIDC_ISSUER" Str]
```

Comment:
what do you think about creating a new integrant definition eg: `:gpml.handler.auth-details` for auth0 details as far as it's currently used (so far) in 2 config places?

Reply:
Sounds good! I see that there is already a const name `:gpml.config/mailjet`, maybe we could use the same name pattern `:gpml.config/auth0`?

Reply:
great!